Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2886,26 +2886,29 @@ - (void)yapDatabaseModified:(NSNotification *)notification
     // b) is inserting new interactions.
     __block BOOL scrollToBottom = wasAtBottom;
 
-    BOOL shouldAnimateUpdates = [self shouldAnimateRowUpdates:rowChanges oldViewItemCount:oldViewItemCount];
-
     void (^batchUpdates)(void) = ^{
         for (YapDatabaseViewRowChange *rowChange in rowChanges) {
             switch (rowChange.type) {
                 case YapDatabaseViewChangeDelete: {
-                    DDLogVerbose(@"YapDatabaseViewChangeDelete: %@, %@", rowChange.collectionKey, rowChange.indexPath);
+                    DDLogVerbose(@"YapDatabaseViewChangeDelete: %@, %@, %zd",
+                        rowChange.collectionKey,
+                        rowChange.indexPath,
+                        rowChange.finalIndex);
```

Comment:
It turns out that `finalIndex` is the correct way to determine the index row of this item after all of the batch updates are complete.